_attributes/common-attributes.adoc

```diff
@@ -187,6 +187,9 @@ endif::[]
 :osdk_ver: 1.31.0
 //Operator SDK version that shipped with the previous OCP 4.x release
 :osdk_ver_n1: 1.28.0
+//Next-gen (OCP 4.13+) Operator Lifecycle Manager, aka "v1"
+:olmv1: OLM 1.0
+:olmv1-first: Operator Lifecycle Manager (OLM) 1.0
```

Comment:
I am borrowing this from the unmerged `main` PR https://github.com/openshift/openshift-docs/pull/65963/files. Whichever merges first, this should come out in the rebase-wash.